pkg/helm/cluster_test.go

```diff
@@ -82,3 +83,75 @@ func TestHelm_OverrideDesiredVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestHelm_filterIgnoredReleases(t *testing.T) {
+	tests := []struct {
+		name              string             // Name of test case
+		releaseIgnoreList []string           // List of release names to be ignored
+		chartIgnoreList   []string           // List of charts to be ignored
+		input             []*release.Release // Input to filtering function
+		want              []*release.Release // Output from filtering function
+	}{
+		{
+			name:              "EmptyInput",
+			releaseIgnoreList: []string{},
+			chartIgnoreList:   []string{},
+			input:             []*release.Release{},
+			want:              []*release.Release{},
+		},
+		{
+			name:              "NoIgnoredReleasesOrCharts",
+			releaseIgnoreList: []string{},
+			chartIgnoreList:   []string{},
+			input: []*release.Release{
+				{
+					Name: "foo",
+				},
+			},
+			want: []*release.Release{
+				{
+					Name: "foo",
+				},
+			},
+		},
+		{
+			name: "AllIgnoredReleases",
+			releaseIgnoreList: []string{
+				"foo",
+			},
+			chartIgnoreList: []string{},
+			input: []*release.Release{
+				{
+					Name: "foo",
+				},
+			},
+			want: []*release.Release{},
+		},
+		{
+			name: "SomeIgnoredReleases",
+			releaseIgnoreList: []string{
+				"foo",
+			},
+			chartIgnoreList: []string{},
+			input: []*release.Release{
+				{
+					Name: "foo",
+				},
+				{
+					Name: "bar",
+				},
+			},
+			want: []*release.Release{{
+				Name: "bar",
+			}},
+		},
```

Comment:
Can you please add a test case here for chartIgnoreList? (see previous comments)

Reply:
Added!